mobile/callog/status_request_log.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/status-im/status-go/internal/sentry"
 )
 
+const placeholder = "***"
```

Comment:
A replacement I've seen is `REDACTED` in uppercase or even `<REDACTED>`.

Reply:
It was actually me who proposed `***` and @qfrank used it.
I don't feel super strong about it, I guess it'a a matter of taste, I don't see any objective difference.

Reply:
The only marginal upside from using something like `REDACTED` is that you can easily and more precisely search for it in logs and that it stands out, whereas `***` requires some guessing as what it means. But totally agree with you, shared just in case.